app/resources/tests.py

```diff
@@ -0,0 +1,85 @@
+from flask_restful import abort, fields, marshal_with, reqparse, Resource, request
+from common.db import execute_sql_query, select_table
+
+test_fields = {
+  'ID': fields.String,
+  'Name': fields.String,
+  'MeasureOf': fields.String,
+  'LevelOfUser': fields.String
+}
+
+class Tests(Resource):
+  @marshal_with(test_fields)
+  def get(self):
+    """
+      Get all tests
+      ---
+      responses:
+        201:
+          description: A list of user items
```

Comment:
change "user items" to "test items"

Reply:
also, maybe this function should accept filters too, in order to search for tests with specific "measures" or "age" etc.